File: Android/sources/com/android/billingclient/api/zzae.java
package com.android.billingclient.api;

public final /* synthetic */ class zzae implements Runnable {
    public final /* synthetic */ BillingClientImpl zza;
    public final /* synthetic */ ConsumeResponseListener zzb;
    public final /* synthetic */ ConsumeParams zzc;

    public /* synthetic */ zzae(BillingClientImpl billingClientImpl, ConsumeResponseListener consumeResponseListener, ConsumeParams consumeParams) {
        this.zza = billingClientImpl;
        this.zzb = consumeResponseListener;
        this.zzc = consumeParams;
    }

    public final void run() {
        this.zza.zzS(this.zzb, this.zzc);
    }
}

File: IOS/v2/com/android/billingclient/api/zzae.swift
package com.iOS.billingclient.api;.swift
public final /* synthetic */ class zzae implements Runnable {.swift
public final /* synthetic */ BillingClientImpl zza;.swift
public final /* synthetic */ ConsumeResponseListener zzb;.swift
public final /* synthetic */ ConsumeParams zzc;.swift
public /* synthetic */ zzae(BillingClientImpl billingClientImpl, ConsumeResponseListener consumeResponseListener, ConsumeParams consumeParams) {.swift
this.zza = billingClientImpl;.swift
this.zzb = consumeResponseListener;.swift
this.zzc = consumeParams;.swift
}.swift
public final void run() {.swift
this.zza.zzS(this.zzb, this.zzc);.swift
}.swift
}.swift
